1주차/12903_목/12903_박경선.java

```diff
@@ -0,0 +1,10 @@
+class Solution {
+    public String solution(String s) {
+        int idx = s.length() / 2;
+        if(s.length() % 2 == 0){
+            return s.substring(idx-1, idx+1);
+        }else{
+            return String.valueOf(s.charAt(idx));
+        }
+    }
```

Comment:
```suggestion
    public String solution(String s) {
        int idx = s.length() / 2;
        if(s.length() % 2 == 0){
            return s.substring(idx-1, idx+1);
        }
        
        return String.valueOf(s.charAt(idx));
    }
```

개인적인 의견이지만 if-else를 쓰지않고 early-return 하는 코드가 더 가독성이 좋다고 생각됩니다.
싹님의 생각이 궁금하네용

Reply:
오오~~! early-return 방법이 훨신 가독성이 좋네여!
개인적인 의견 넘 감사드려요~ 많이많이 해주셔도 좋습니다 😃